src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/CNVInputReader.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.vcf.VCFHeader;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.engine.FeatureDataSource;
+import org.broadinstitute.hellbender.utils.Utils;
+import scala.Tuple2;
+
+import java.util.ArrayList;
+
+public class CNVInputReader {
+
+    public static Broadcast<SVIntervalTree<VariantContext>> broadcastCNVCalls(final JavaSparkContext ctx,
```

Comment:
I'd still sort of prefer for this method to be on one of the tool classes I think. I think the `Reader` class should just read the calls, validate them, and put them into the interval tree. Then the tool can broadcast them. That way the tool class is the only one that really has to worry about the Spark mechanics of this. Does that make sense to you?

Reply:
Makes sense. 
Just to make sure I understand the request: it is OK to have a 
`Broadcast<SVIntervalTree<VariantContext>> cnvCallsBroadcast` in `SvDiscoveryInputData`, just to move the broadcasting method into an actual tool. Am I understanding correctly?

Reply:
now refactored